lib/generators/hyrax/assets_generator.rb

```diff
@@ -18,11 +18,8 @@ def inject_css
 
   def inject_js
     return if hyrax_javascript_installed?
-    insert_into_file 'app/assets/javascripts/application.js', after: '//= require_tree .' do
-      <<-JS.strip_heredoc
-
-        //= require hyrax
-      JS
+    insert_into_file 'app/assets/javascripts/application.js', after: "//= require blacklight/blacklight\n" do
```

Comment:
With the generation of a new app using Rails 6, the `application.js` generated is missing `//= require_tree .`, which led to the failure of the inseration of the Hyrax js. This change insures the inclusion of `hyrax` is inserted below `blacklight/blacklight`, which is a core dependency that will always be included when the test app is generated.